packages/assets-controllers/src/RatesController/multichain-rates/util.ts

```diff
@@ -0,0 +1,21 @@
+import { handleFetch } from '@metamask/controller-utils';
+
+import type { MarketDataDetailsFromPriceAPi } from './types';
+
+/**
+ * Fetches the rate for a given token on a specific chain.
+ *
+ * @param assetIds - An array of addresses of the tokens to fetch the rate for.
+ * @param includeMarketData - Whether to include market data in the response.
+ * @returns A promise that resolves to the rate of the token in USD.
+ */
+export function fetchMultiChainRates(
+  assetIds: string[],
+  includeMarketData: boolean,
+): Promise<MarketDataDetailsFromPriceAPi> {
+  const url = `http://localhost:3000/v3/spot-prices?assetIds=${assetIds.join(
+    ',',
+  )}${includeMarketData && `&includeMarketData=true`}${`vsCurrency=usd`}`;
```

Comment:
Does `vsCurrency=usd` need an `&`?  `&vsCurrency=usd`?

Reply:
Also, I don't think we need the `${}` for `vsCurrency=usd`

Reply:
the api call will be replaced by the snap call below once this one will be ready

https://github.com/MetaMask/SIPs/pull/154/files#diff-d13ba923939d83e166cd51c5051566c06e60068d79d2ad2897f951df66847fe8R96-R106

for now i'm just using a call to a locallfake server, but we will not have api call anymore